GeometryCore/include/Polygon.h

```diff
@@ -91,12 +91,16 @@ namespace geom_utils
         std::vector<Triangle2D> triangulate() const;
 
         static Polygon makePolygon(const Triangle2D& tri);
+
+        //return inflate poligon
```

Comment:
That comment is too obvious. It's understandable from the `inflate` method signature that it returns polygon